src/rln.ts

```diff
@@ -158,12 +162,39 @@ export function sha256(input: Uint8Array): Uint8Array {
   return zerokitRLN.hash(lenPrefixedData);
 }
 
+type StartRLNOptions = {
+  /**
+   * If not set - will extract MetaMask account and get provider from it.
+   */
+  provider?: ethers.providers.Web3Provider;
```

Comment:
nit: unless there is a specific reason to use `Web3Provider` we can just use `Provider`